libs/ui-watt/src/lib/components/dropdown/+storybook/storybook-dropdown-reactive-forms.stories.ts

```diff
@@ -101,6 +104,36 @@ SingleSelect.parameters = {
   },
 };
 
+export const SingleSelectChipMode: StoryFn<WattDropdownComponent> = (
```

Comment:
Suggestion: Could be reuse the WattDropDownComponent stories we already have. But pass chipMode to them?

Reply:
Good point. I will look into that 👍